gaussopt: corrected the definition of BeamParameter.radius

operte opened the pull request:
BeamParameter.radius was defined incorrectly. Previously it was
defined as self.z_(1 + (self.z/self.z_r)__2), but it should be
self.z_(1 + (self.z_r/self.z)**2).

operte committed:
gaussopt: corrected the definition of BeamParameter.radius

BeamParameter.radius was defined incorrectly. Previously it was
defined as self.z*(1 + (self.z/self.z_r)**2), but it should be
self.z*(1 + (self.z_r/self.z)**2).

jksuom commented:
It would be useful to have a reference to the wikipedia article on the Gaussian beam in the docstring of ``BeamParemeter`` in addition to the one on the complex beam parameter. That article contains the correct formula for ``radius``.

jksuom commented:
(You can simply add a new commit to the branch and push it to github.)

jksuom commented:
There is also a docstring in ``radius``. You should put the corrected value in the example and verify it by bin/doctest (or bin/doctest gaussopt).

operte committed:
gaussopt: updated the docstrings of BeamParameter and radius

Added a new reference [2] to the docstring of BeamParameter with a
correct definition of a gaussian beam's radius.
Updated the output of the example in the docstring of
BeamParameter.radius according to the correct definition of a
gaussian beam's radius.

operte commented:
Thanks for your help! I hadn't thought of updating the docstring. Newbie mistakes :)

jksuom commented:
There are also tests in test_gaussopt.py. [This line](https://github.com/sympy/sympy/blob/master/sympy/physics/optics/tests/test_gaussopt.py#L86) should be changed. You can verify the result by running bin/test gaussopt.

operte committed:
gaussopt: test_gaussopt now correctly tests BeamParameter.radius

Updated sympy.physics.optics.tests.test_gaussopt.py to test
sympy.physics.optics.gaussopt.BeamParameter.radius according to its
correct definition.

operte commented:
Done ;)

jksuom commented:
Thanks! Merging.
